Dockerfile

```diff
@@ -6,12 +6,12 @@ LABEL maintainer="BOSCoin Developers <devteam@boscoin.io>"
 COPY ./ /go/src/boscoin.io/sebak
 WORKDIR /go/src/boscoin.io/sebak
 
-## Since those need to be re-run every time anyway, they are in a single stage
-# `git` and `openssh` are needed for `go get`
-RUN apk add --no-cache git openssh          \
-    && go get github.com/golang/dep/cmd/dep \
-    && dep ensure -v                        \
-    && go install -v ./...
+## Note that we do not get the dependencies anew
+## We carry over whatever is in `vendor`, so the user MUST run `dep ensure` in their local copy
+## This make building the container orders of magnitude faster (`dep ensure` is extremely slow),
+## greatly reduce the container's size, and gives more control to the user as to what is tested
+## (one can replace a dependency, if needed).
```

Comment:
I'm not sure about this.
What people expected when they see Dockerfile, `docker build ...` should be just worked.
Some pre-process is a little bit confused.

Additionally, why container's size reduced even if we use docker multi-staged build?

Reply:
I don't have a current solution, but...
Our problem is our dependencies are too large. So, our direction should be reducing our dependencies. If this approach is temporary, it is ok. However, I'm worrying about we forget the dependencies issue if this mitigates the docker build time. Docker's build time is not a critical issue for now.
And I don't know `dep` mechanism much as well.

Reply:
> Docker's build time is not a critical issue for now.

It makes the edit-compile-run cycle very slow because the time spent `dep ensure`ing dominates by at least an order of magnitude, which is why it came as a problem to me.

Another approach I thought of this weekend would be to use volumes and mount the `vendor` directory, so that `dep ensure` cache is kept. Not sure that's the best solution though. Are we fine with merging this as a temporary solution ?